docs/stylesheets/alcf-extra.css

```diff
@@ -210,9 +210,9 @@ body {
 	margin-top: 6px;
 }
 
-#nav-sup {
-	padding-left: 32px;
-}
+/* #nav-sup { */
+/* 	padding-left: 32px; */
+/* } */
```

Comment:
I'm not sure who wrote this originally so I left the previous CSS rules as comments ??

Reply:
@sbxchicago perhaps? Are we confident that removal doesnt mess things up on all browsers, platforms, etc?

Reply:
I believe it should be, it looks like the navbar is condensed and simplified for smaller devices, e.g.

- from iPad:

  ![ScreenShot-2024-11-08-001659@2x](https://github.com/user-attachments/assets/8bc310f9-206e-4fb4-9e06-51cd17737eba)
  ![ScreenShot-2024-11-08-001714@2x](https://github.com/user-attachments/assets/43fb4d7d-87e8-443b-b11b-3bb910f2ca0d)

- from iPhone:
  - Left: <https://docs.alcf.anl.gov> from iPhone
  - Right: serving with these changes from localhost 

  <img width="25%" src="https://github.com/user-attachments/assets/66be62f3-0b88-47a4-85d1-942fa539fa38">
  <img width="25%" src="https://github.com/user-attachments/assets/f7dd3d12-468c-41af-a625-1af5c6b99531">